cmd/namespace/use.go

```diff
@@ -127,7 +127,7 @@ func getNamespacesSelection(ctx context.Context) ([]utils.SelectorItem, error) {
 
 	namespaces := []utils.SelectorItem{}
 	for _, space := range spaces {
-		if space.Status == "Active" {
+		if space.Status != "Deleting" {
```

Comment:
What are the possible status here? It seems like this has the potential to begin showing namespaces that weren't show before?

Reply:
@jmacelroy Potential statuses here are:  `Active`,  `DestroyAllFailed`, `DestroyingAll`,  `Deleting`,  `Sleeping` and `Inactive` (This is deprecated but may happen that some space still returns this status. CLI converts it to `Sleeping`.

Reply:
Should I then change the if to `space.Status == "Active" || "Sleeping"`? Or would the current one (`space.Status != "Deleting"`) suffice?

Reply:
It depends on what we want. With the current condition, if a new status is added to spaces, those spaces would automatically be listed. With the condition you are proposing, they won't appear. 

I think current condition is ok, because the other statuses (`DestroAllFailed` and `DestroyingAll`) are active spaces where the user can deploy new things, but not sure what status is displayed in the list. Can you try to execute a `Destroy All` operation from the UI and get the list to see how it looks like?

Reply:
Should we get this status somehow with constants so they keep somewhere listed and refer to them as constants?

Reply:
> Can you try to execute a Destroy All operation from the UI and get the list to see how it looks like?

By "get the list" you mean using the `space.Status != "Deleting"` condition right?

> Should we get this status somehow with constants so they keep somewhere listed and refer to them as constants?

Do you mean defining constants for each of the possible statuses?

Reply:
@RinkiyaKeDad yes, I meant using that condition to see how that state is reflected in the output

Reply:
@ifbyol I tried that:


https://user-images.githubusercontent.com/56963264/154237098-68e373b6-6f8c-408a-a48a-11917d119cc9.mov



So the namespace gets shown even if we click "Destroy All" in the UI. This is expected right? Since we're deleting only the deployed application resources and not the namespace itself.

Reply:
Yeah, that is expected. I just wanted to make sure those ns also appear in the list, thanks for testing it!

Reply:
@RinkiyaKeDad  @rberrelleza @ifbyol I was thinking about a new approach here. It could be needed to point to a `Deleting` namespace, for example, to remove its finalizers.
Instead of ignoring them, why don't we show the namespace status as part of the namespace selector?

Reply:
I think the Okteto service account doesn't have permissions to patch a namespace, at least I wasn't able

```
kubectl edit ns nacho
error: namespaces "nacho" could not be patched: namespaces "nacho" is forbidden: User "system:serviceaccount:okteto:e9492a1e-e86a-4cf2-bb63-90557e2a4a85" cannot patch resource "namespaces" in API group "" in the namespace "nacho"
```

Although they are still be able to patch any resource within it to remove their finalizers.

It could be an option, but not sure if it is the best experience. If we let a user to select a deleting namespace, we could leave them pointing to a non-valid context if the namespace is finally deleted. Finalizers issue should not be a common scenario and I think we should provide another solution for that at product level instead of leaving the responsibility to the user

Reply:
Let's merge it as it's and we can iterate later